tests/acceptance/blocks/page.js

```diff
@@ -0,0 +1,16 @@
+import { Selector, t } from 'testcafe';
+
+/**
+ * Models the user interaction with the page as a whole
+ */
+class Page {
```

Comment:
If we have a base `Page` class like this, what do you think about having all the other, specific `Pages` extend it?

Reply:
Oliver and I opted to not create specific pages like we do in the SDK for the theme acceptance tests. When writing the new theme acceptance tests, Oliver asked me to make them simpler, and my solution was to remove the Page abstraction from the tests. By removing that abstraction the tests are simpler because they no longer require getters and setter to access the blocks from each page. I looked into testcafe's guides for writing tests with a pages model, and I think that makes a lot of sense for websites with completely different functionality on each page, but since the search ui is component based, organizing the code into blocks which correspond with those components is simpler. The acceptance tests in the SDK have an additional layer of abstraction that not even the testcafe page model examples have, and I think that these tests are actually more similar to testcafe's examples in terms of the levels of abstraction used

Reply:
Got it, thanks for the explanation! I was assuming a similar abstraction layer to the SDK. Let's see how this level of abstraction works for us in the Theme. So far it does seem to make things simpler! If all goes well, maybe we change the SDK to use this scheme as well.